gnomad/assessment/summary_stats.py

```diff
@@ -355,6 +354,181 @@ def get_tx_expression_expr(
     )
 
 
+def get_summary_stats_variant_filter_expr(
+    t: Union[hl.Table, hl.MatrixTable],
+    filter_lcr: bool = False,
+    filter_expr: hl.expr.SetExpression = None,
```

Comment:
I don't think that the longer name is needed, we often use `filter_expr` in our code, and in this context I don't see how it would be confused with the function name

Reply:
Okay, I was looking at gnomad_qc, where you had filter_exprs={...} but you had a note there. So fine with me.